.eslintrc.js

```diff
@@ -3,7 +3,7 @@ const config = {
     browser: true,
     es2021: true,
   },
-  extends: ['airbnb-base'],
+  extends: ["airbnb-base"],
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/TheCodeCartel/mergify-test-service/pullRequest?prid=7839457)
```suggestion
  extends: ['airbnb-base'],
```